apollo-router-core/src/response.rs

```diff
@@ -39,6 +40,19 @@ pub struct Response {
     pub extensions: Object,
 }
 
+/// temporary structure to help deserializing errors
+#[derive(Deserialize)]
+struct ResponseMeta {
+    #[serde(skip_serializing_if = "Option::is_none", default)]
+    label: Option<String>,
+
+    #[serde(skip_serializing_if = "Option::is_none", default)]
+    path: Option<Path>,
+
+    #[serde(skip_serializing_if = "Option::is_none", default)]
+    has_next: Option<bool>,
+}
```

Comment:
:grey_question: oh same thing here. There are even "skip_serializing_if" attributes despite the struct does not implement Serialize?

Reply:
right, I should just remove those

Reply:
removed in 62ccdc9